specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/examples/HanaInstances_Get.json

```diff
@@ -18,6 +18,7 @@
                 "properties": {
                     "hanaInstanceId": "00000000-0000-0000-0000-000000000000",
                     "powerState": "restarting",
+                    "proximityPlacementGroup": "testPPG",
```

Comment:
this should be a full ARM path

Reply:
You mean something like this:
`/subscriptions/b7079faf-f1ba-42fb-97e8-7fcf3b2f7cb3/resourcegroups/NA_POC_SJC_1-T050/providers/Microsoft.Compute/networkPlacementGroups/myplacementgroup
`

Reply:
Hiding actual ids is generally a good practice

Reply:
@dwj300 could you explain what a full ARM path would look like.

Reply:
@dsgouda  agreed that we should hide the actual id, @dwj300 and I talked about it, it would look something like this:
`/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.Compute/proximityPlacementGroups/myplacementgroup`